src/crypto/shelley/transactionUtils.js

```diff
@@ -74,7 +74,8 @@ export const createUnsignedTx = async (
         absSlotNumber,
         protocolParams,
         [],
-        // [], // TODO
+        [],
+        true,
```

Comment:
did you decide you want to allow 0-output transactions?

Reply:
Ah good point, no not definitely. But I think I'll just keep it consistent with the extension's behaviour.

Reply:
Okay. Just FYI the extension disabled 0-output transactions in all cases now.